net-news/sfeed/sfeed-1.2.ebuild

```diff
@@ -12,29 +12,33 @@ SRC_URI="https://codemadness.org/releases/${PN}/${P}.tar.gz"
 LICENSE="ISC"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE="ncurses"
-
-DEPEND="ncurses? ( sys-libs/ncurses )"
-RDEPEND="${DEPEND}
-	net-misc/curl
-	sys-apps/coreutils
-	sys-libs/glibc
-	virtual/awk
-	www-client/lynx
-	x11-misc/xclip
-	x11-misc/xdg-utils"
-BDEPEND=""
-
-PATCHES=(
-	"${FILESDIR}/${PN}-1.2-ldflags.patch"
-)
+
+THEMES=( mono{,-highlight} newsboat templeos )
+IUSE="+ncurses +$(printf "theme-%s " ${THEMES[@]})"
+REQUIRED_USE="ncurses? ( ^^ ( $(printf "theme-%s " ${THEMES[@]}) ) )"
+
+DEPEND="ncurses? ( sys-libs/ncurses:= )"
+RDEPEND="${DEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_configure() {
+	for name in "${THEMES[@]}"; do
+		if use theme-${name}; then
+			SFEED_THEME="${name//-/_}"
+		fi
+	done
+
+	restore_config $(printf "themes/%s.h " ${THEMES[@]//-/_})
+}
 
 src_compile() {
 	emake \
 		CC=$(tc-getCC) \
 		RANLIB=$(tc-getRANLIB) \
 		AR=$(tc-getAR) \
-		SFEED_CURSES=$(usex ncurses "sfeed_curses" "")
```

Comment:
Unrelated to the changes here, but the CC/RANLIB/AR above this line are lacking quotes

if for example had CC="gcc -m32" (or anything else, but -m32 is a common case) it'll fail to build.